File: SEM-3/dsa sem 3/Practical 3/program2.c
#include <stdio.h>
struct employee{
    char Name[10];
    int salary;
};

void main(){
    struct employee e1;
    printf("Enter Name of Employee:");
    getchar();
    scanf("%s",e1.Name);
    printf("Enter Salaray of Employee: ");
    scanf("%d",&e1.salary);
    printf("Name of the Employee : %s",e1.Name);
    printf("\nSalary of the Employee : %d",e1.salary);   
}

File: SEM-3/Python/practical 3/program2.py
#2)(a)
i=0
L=[]
while(i<3):
    BS=int(input("Enter Basic Salary of Employee :"))
    L.append(BS)
    i+=1
i=0
while(i<3):
    BS=L[i]
    HRA=(5*100)/BS
    DA=(20*100)/BS
    MA=(10*100)/BS
    GS=BS+HRA+DA+MA
    print("Gross Salary of Employee",i,"is Rs",GS)
    i+=1